README.md

```diff
@@ -230,6 +230,34 @@ the `@optional` does not apply, and that value is then checked against the filte
 coercion. These subsequent operations may then cause the result set to be discarded if it does
 not match.
 
+For example, suppose we have two Person nodes with names `Albert` and `Betty` such that there's an edge 
+`Albert-knows->Betty`.
```

Comment:
I think it'd be best if we used proper GraphQL terminology here. How about:

For example, suppose we have two `Person` vertices with names `Albert` and `Betty` such that there is a `Person_Knows`  edge from `Albert` to `Betty`.

Reply:
Done-- will be reflected in the next commit (same goes for all the conversations I resolve here before committing, just convenient to keep track of things).